lib/gateway.dart

```diff
@@ -353,15 +353,24 @@ class Gateway {
           officers: organization?.officers,
           parkingLots: lots
         );
+        request = ModelMutations.update(newOrg);
+        response = await Amplify.API.mutate(request: request).response;
+        print(response.data);
+        if (response.errors.isEmpty) {
+          return "Success";
+        }
+        print(response.errors);
       }
       else {
         organization?.parkingLots?.add(lot);
-      }
-      if (newOrg != null) {
-        request = ModelMutations.update(newOrg);
+        request = ModelMutations.update(organization!);
```

Comment:
this will blow up if _organization_ is ever null. Should it never be null in this case?